src/FormattingTrait.php

```diff
@@ -32,7 +32,7 @@ trait FormattingTrait
      */
     public static function resetToStringFormat(): void
     {
-        static::setToStringFormat(Chronos::DEFAULT_TO_STRING_FORMAT);
+        static::setToStringFormat(static::DEFAULT_TO_STRING_FORMAT);
```

Comment:
Before this fix and addition of the constant to `ChronosDate`, calling `ChronosDate::__toString()` after `ChronosDate::resetToStringFormat()` returned value in `Y-m-d H:i:s` format instead of `H:i:s`.